AsyncTask leak/crash in 5.2.6

8acd5f1 introduced `LibraryTask` which crashes the app if an orientation change occurs while the libraries are loading. Stack trace:

```
E/AndroidRuntime: FATAL EXCEPTION: AsyncTask #1
   Process: com.mikepenz.aboutlibraries.sample.debug, PID: 17591
   java.lang.RuntimeException: An error occured while executing doInBackground()
       at android.os.AsyncTask$3.done(AsyncTask.java:300)
       at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:355)
       at java.util.concurrent.FutureTask.setException(FutureTask.java:222)
       at java.util.concurrent.FutureTask.run(FutureTask.java:242)
       at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:231)
       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
       at java.lang.Thread.run(Thread.java:818)
    Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String android.content.Context.getPackageName()' on a null object reference
       at com.mikepenz.aboutlibraries.Libs.getStringResourceByName(Libs.java:490)
       at com.mikepenz.aboutlibraries.ui.LibsFragment.extractBooleanBundleOrResource(LibsFragment.java:249)
       at com.mikepenz.aboutlibraries.ui.LibsFragment.access$100(LibsFragment.java:37)
       at com.mikepenz.aboutlibraries.ui.LibsFragment$LibraryTask.doInBackground(LibsFragment.java:98)
       at com.mikepenz.aboutlibraries.ui.LibsFragment$LibraryTask.doInBackground(LibsFragment.java:71)
       at android.os.AsyncTask$2.call(AsyncTask.java:288)
       at java.util.concurrent.FutureTask.run(FutureTask.java:237)
       at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:231) 
       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112) 
       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587) 
       at java.lang.Thread.run(Thread.java:818) 
```

--- a/library/src/main/java/com/mikepenz/aboutlibraries/ui/LibsFragment.java
+++ b/library/src/main/java/com/mikepenz/aboutlibraries/ui/LibsFragment.java
@@ -44,6 +44,8 @@ public class LibsFragment extends Fragment {
 
     private Comparator<Library> comparator;
 
+    private AsyncTask mLibTask;
+
     /**
      * Default Constructor
      * Gets an libs instance and gets all external libs
@@ -229,7 +231,16 @@ public class LibsFragment extends Fragment {
         //load the data
         if (getContext() != null) {
             //fill the fragment with the content
-            new LibraryTask(getContext()).execute();
+            mLibTask = new LibraryTask(getContext()).execute();
+        }
+    }
+
+    @Override
+    public void onDestroyView() {
+        super.onDestroyView();
+        if (mLibTask != null) {
+            mLibTask.cancel(true);
+            mLibTask = null;
         }
     }
 

